lang/lua-struct/Makefile

```diff
@@ -0,0 +1,36 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lua-struct
+PKG_VERSION:=0.9.2
+PKG_RELEASE:=1
+
+PKG_LICENSE:=MIT/X11
+PKG_URL:=https://github.com/iryont/lua-struct
+
+PKG_MAINTAINER:=jasle <jasle@riseup.net>
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/iryont/$(PKG_NAME)/tar.gz/$(PKG_VERSION)?
```

Comment:
Why do we need to add package, which is no longer developed and also the repository is archived for almost an year, now?

Reply:
`lua-struct` is a dependency of `mt5311`, so it is needed in order to use the tools for the metanoia sfp vdsl modems, which are very useful when using such a modem.

It is just a very small package, in the end just one lua file with about 200 lines. So not much that could be further developed.

If desired we could switch to a fork of the repo for the package, but then it would base on a different source than the package in luarocks.

Reply:
I understand your need of mt5311, but I dont see why we should fork the repo of package, which is dead, no longer developed and I dont think that you will take care of the forked package, anyway. 

You should raise the issue with mt5311, if they can find the replacement or what they will do with it.

Reply:
I've raised the issue with mt5311.

There is no real alternative to using `lua-struct`, as a pure lua package without c bindings and with support for lua 5.1 is desired.

The other struct modules for lua are also kind of old and have no recent releases. Seems like implementing structs for lua is just is not a thing, that need continuously development at all.

Could include `lua-struct` directly into the repository of `mt5311`, but this would just be a dirty hack to avoid having it as external dependency and directly including dependencies into the repository isn't a good way at all for various reasons.

I think we can refrain from the _**should** be maintained_ for this small package here.